adafruit_as7341.py

```diff
@@ -616,7 +615,9 @@ def led_current(self):
     @led_current.setter
     @_low_bank
     def led_current(self, led_curent):
-        new_current = int((led_curent - 4) / 2)
+        """coerce led_curent to be between 4 & 258, then calculate value
+        to be put into register on chip"""
```

Comment:
This comment will not be rendered in the documentation because only the docstring for the getter/`@property` is rendered.  That said, I'm OK with leaving it as a explanation of the code below it. It could also be a comment.

Please edit the setter docstring to mention the range of valid values, and to add something like "invalid values will be rounded to the closest valid number". No need to get overly specific. People who care about the nitty gritty details can read the source ;)

Please keep in mind that we try to talk about properties as nouns and not verbs.  `led_current` doesn't _do_ something, it _is_ something. Apologies if this sounds a bit pedantic, but we try our best to remain consistent in style.

For more info you can see the design guide here:
https://circuitpython.readthedocs.io/en/latest/docs/design_guide.html